web/src/routes/backOffice/employees/list/EmployeesTable.svelte

```diff
@@ -0,0 +1,126 @@
+<script lang="ts">
+	import type {
+		Employee,
+		EmployeeSortableFields,
+	} from "../../../../domain/employees";
+	import Table from "../../../../lib/components/table/Table.svelte";
+	import type {
+		Columns,
+		SortingDirection,
+	} from "../../../../lib/components/table/types";
+	import { DEFAULT_PAGE_SIZE } from "../../../../lib/constants/pagination";
+	import { t } from "../../../../lib/utils/i8n";
+	import { rolesOptions } from "../constants/roles";
+	import employeesStore from "./employeesStore";
+	import { formatDate } from "../../../../lib/utils/date";
+	import { DateFormats } from "../../../../lib/constants/date";
+
+	const { loading, data, filters } = employeesStore;
+
+	const columns: Columns<Employee> = [
+		{
+			type: "accessor",
+			field: "firstName",
+			header: $t("employees.name"),
+			enableSorting: false,
+			enableFiltering: false,
+			cell(firstName, row) {
+				return `${firstName} ${row.lastName}`;
+			},
+		},
+		{
+			type: "accessor",
+			field: "username",
+			header: $t("employees.username"),
+			enableSorting: true,
+			enableFiltering: false,
+			cell(username) {
+				return username;
+			},
+		},
+		{
+			type: "accessor",
+			field: "role",
+			header: $t("employees.role"),
+			enableSorting: false,
+			enableFiltering: true,
+			filterOptions: rolesOptions.map(role => {
+				return {
+					value: role,
+					label: $t(`employees.role.${role}`),
+				};
+			}),
+			filterInitialValue: $filters.role,
+			cell(role) {
+				return $t(`employees.role.${role}`);
+			},
+			onFilterChange(role) {
+				filters.update(filters => {
+					return {
+						...filters,
+						pageIndex: 0,
+						role,
+					};
+				});
+			},
+		},
+		{
+			type: "accessor",
+			field: "scheduleStart",
```

Comment:
This column actual shows both `scheduleStart` and `scheduleEnd`.
Can't we rename this field to just `schedule`?
```suggestion
			field: "schedule",
```